src/styles/StyleUtils.js

```diff
@@ -375,6 +375,19 @@ function getMiniReportActionContextMenuWrapperStyle(isReportActionItemGrouped) {
     };
 }
 
+/**
+ * Gives the minWidth for Resend Link Button
+ *
+ * @param {String} currentLocale
+ * @returns {Object}
+ */
+function getResendLinkStyle(currentLocale) {
+    return {
+        minWidth: currentLocale === CONST.DEFAULT_LOCALE ? 108 : 139,
```

Comment:
Would you mind explaining why it's best to hardcode these widths based on the locale? If we add french for example, would this need to be updated since there are now more than 2 choices?

Reply:
Yeah, I think a better approach would be using get width value from CONST based on locale like this
(the keys could be all uppercase as well if we want)
```js
RESEND_LINK_WIDTH:{
     "en": "xxx",
     "es": "xxx",
     "xx": "xxx"
}

```

like for emojiPicker a similar const is being used
https://github.com/Expensify/App/blob/51de5d9ea050fa83a9ec0286e1806f51a1e20876/src/CONST.js#L347

Reply:
Is there a way to dynamically get the width of the button once text is in it? I'm just worried that this means if we ever change the copy, the developer will need to know to also change this style

Reply:
I was trying a few things but was unable to get the results. So I think if we ever change the copy then we have to update the widths as well

Reply:
What if we just set the minimum to the larger one (139) and then if we ever add something longer we can make the button bigger. @shawnborton do you think it would look bad if the button has some blank space on the sides around the text rather than sizing it based on the text?

Reply:
I'm fine with taking that approach. It feels like this could be as simple as just having a generic width class somewhere, and then passing that in as an extra style to the button.

Reply:
Sweet. Want to do that then @thesahindia ?

Reply:
@stitesExpensify, done.